tensorflow/java/maven/pom.xml

```diff
@@ -32,6 +32,8 @@
     <module>libtensorflow_jni_gpu</module>
     <module>tensorflow</module>
     <module>proto</module>
+    <module>tensorflow-hadoop</module>
```

Comment:
Should we make these names shorter, they are already in the `org.tensorflow` group, so:

- `org.tensorflow:hadoop` might be better than `org.tensorflow:tensorflow-hadoop`, and
- `org.tensorflow:spark-connector` might be better than `org.tensorflow:spark-tensorflow-connector'
?

Reply:
Great suggestion. I'll make the changes.